e2e/gui/src/test/java/com/epam/pipeline/autotests/SystemLoggingTest.java

```diff
@@ -90,12 +90,12 @@ public void userAuthentication() {
             if (impersonateMode()) {
                 systemLogsAO.filterByMessage("impersonation");
             }
-            adminInfo = getInfo(systemLogsAO, format("Successfully authenticate user: %s", admin.login.toUpperCase()),
+            adminInfo = getInfo(systemLogsAO, format("Successfully authenticate user: %s", "NotAuthorized"),
```

Comment:
Why did we change this to "NotAuthorized"?

Reply:
Changes have been removed.